inst/@sym/private/mat_access.m

```diff
@@ -1,4 +1,5 @@
-%% Copyright (C) 2014 Colin B. Macdonald
+%% Copyright (C) 2014-2016 Colin B. Macdonald
+%% Copyright (C) 2016 Abhinav Tripathi
```

Comment:
I list copyright for years where the file was actually modified.  So I would leave this as "2014 Colin..." because I didn't modify in 2015 and made only trivial changes in 2016.